app/controllers/sessions_controller.rb

```diff
@@ -76,20 +80,22 @@ def magic_link_sent; end
 
   def signout
     if current_user.present?
-      if development_mode_auth? || magic_link_enabled?
-        # Disappointingly, with HTTP basic auth it's trick to really log
-        # someone out, since the browser just holds onto the user's username /
-        # password and re-submits it until their session ends. So they'll just
-        # create a new session after this "logout".
+      case session[:auth_user]["provider"]
+      when "developer"
         reset_session
-        redirect_to root_path
+        redirect_to "/personas"
       else
-        uri = URI("#{Settings.dfe_signin.issuer}/session/end")
-        uri.query = {
-          id_token_hint: current_user["credentials"]["id_token"],
-          post_logout_redirect_uri: "#{Settings.dfe_signin.base_url}/auth/dfe/signout",
-        }.to_query
-        redirect_to uri.to_s
+        if magic_link_enabled?
+          reset_session
+          redirect_to root_path
```

Comment:
test coverage?

Reply:
this path should be identical to how it was. I can add the missing coverage for this code

Reply:
Coverage for this path has now been added